### DISCUSSION TITLE
Can we have an option to delete the duplicate

### DISCUSSION THREAD

**Discussion by DareDevil2000** [Authenticator-Extension/Authenticator#1198] [Ideas]
As the title says. 
Why is it necessary? It is necessary when using two different browsers and wishing to have all the accounts added in them. However, upon importing from another browser, the existing accounts are also added, resulting in a duplication.

**Comment by Spiffo** [comment]
That is exactly what I came here for. Using multiple browsers + Export - Import causes duplication. Either prevent these from being added, or request to "add / update" or "truncate and add".

**Reply by Spiffo** [reply]
P.S. : https://github.com/Authenticator-Extension/Authenticator/discussions/813 shows how to clear before importing, but it's not really clear.
